platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -21,112 +21,96 @@ describing the following:
 
 ## Why are platforms valuable
 
-What economic and business reasons motivate enterprises to implement platforms?
-The value of platforms emerges from a) the consolidation of shared capabilities
-in a team of experts focused on b) making those services and capabilities easy
-to integrate and use in digital products and applications.
+What economic and business reasons motivate enterprises to implement platforms
+for cloud-native computing?  The value of platforms emerges from a) the
+consolidation of shared capabilities in a team of experts focused on b) making
+those services and capabilities easy to integrate and use in digital products
+and applications.
 
 Enterprises adopting platform engineering can expect to achieve the following:
 
 1. Accelerate product development and delivery by intentionally dedicating teams
-   to platform capabilities and thereby reducing cognitive load on product teams
+   to platform services and thereby reducing cognitive load on product teams
 1. Improve reliability and resiliency of products relying on platform
    capabilities by dedicating experts to configure and manage them
 1. Accelerate product development and delivery by reusing and sharing platform
    tools and knowledge across many teams in an enterprise
 1. Reduce risk of security, regulatory and functional issues in products and
    services by governing platform capabilities and the users, tools and processes
    surrounding them
-1. Enable cost-effective and productive use of capabilities from public clouds
+1. Enable cost-effective and productive use of services from public clouds
    and other providers by enabling delegation of implementations to those providers
    while maintaining control over user experience
 
 These benefits accrue in part because just a few platform teams serve many
 product teams, multiplying their impact; in part because platform teams
-consolidate management of common capabilities, facilitating governance; and in
+consolidate management of common functionality, facilitating governance; and in
 part because platform teams emphasize user interfaces and experiences above all
 else.
 
-A team of platform experts not only reduces common work [1] demanded of product
-teams but also optimizes platform capabilities [2] for those products. And
-because a platform offers patterns for many capabilities, knowledge and tools
-for those spread broadly [3] and developers can quickly contribute to other
-teams and products built on the same foundation. Governance and controls [4] can
-also be embedded in a platform's patterns and capabilities. Finally, because
-platform teams corral capability providers and provide consistent experiences
-over their offerings, they enable efficient use of public clouds and service
-providers [5] for foundational but undifferentiated capabilities like databases
-and identity.
+A team of platform experts not only reduces common work \[1\] demanded of product
```

Comment:
Are these brackets supposed to be escaped? I saw a pattern of double brackets above.

Reply:
Yes, they reference the bullet points in the list above. They're in there to keep us editors honest at least and be sure we discuss all the bullet points in the prose. We can remove them in the final, or clarify them.